src/beautifiers/clojure-beautifier/index.coffee

```diff
@@ -0,0 +1,22 @@
+"use strict"
+path = require('path')
+Beautifier = require('../beautifier')
+
+module.exports = class ClojureBeautifier extends Beautifier
+
+  name: "Clojure Beautifier"
```

Comment:
I'd prefer the name be `cljfmt`, instead of generic `Clojure Beautifier`. As you may have seen, some of these languages have multiple beautifiers, so using `<Language> Beautifier` would not work 😉.